### PR TITLE
Adds QM as a job 

### PR DESCRIPTION
**Pull request by bardinatavern4**
Compano asked for it, it compiles fine, appears to work, everything seems good.

### CODE REVIEW

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -201,6 +201,43 @@ Medic
...
+/datum/outfit/job/ncr/f13quartermaster
+	name = "NCR Quartermaster"
+	jobtype = /datum/job/ncr/f13quartermaster
+	id = 			/obj/item/card/id/ncrquartermaster
+	uniform =  		/obj/item/clothing/under/f13/ncr
+	accessory =     /obj/item/clothing/accessory/ncr/ssgt
+	suit = 			/obj/item/clothing/suit/armor/f13/ncrarmor/mantle
+	head = 			/obj/item/clothing/head/f13/ncr
+	glasses = 		/obj/item/clothing/glasses/sunglasses/big
+	suit_store  = /obj/item/gun/ballistic/automatic/marksman
```
Comment (Karl-Franz-I) [inline code/modules/jobs/job_types/ncr.dm]: Might wanna not give them one of the best rifles in the game, that doesn't spawn on the map, and is reserved for adminspawn Legate/Colonel.

Comment (RenGusta) [reply code/modules/jobs/job_types/ncr.dm]: Definitely want to consider that point. Are you purging a sergeant slot to compensate?

Comment (bardinatavern4) [reply code/modules/jobs/job_types/ncr.dm]: did so but RenGusta (The guy above) after i removed a sergeant slot and gave them a shotgun decided to not add it and overrule compano.

**Comment by Hekzder** [comment]
PR seems mostly abandoned, along with merge conflicts and QM memes. There is nothing preventing a Sergeant or higher from RPing a QM and it happens occasionally, it would happen more often if a QM was actually important.

**Pull request closed without merging**